casper/src/main/scala/coop/rchain/casper/rholang/RuntimeManager.scala

```diff
@@ -39,25 +40,24 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.FiniteDuration
 
 trait RuntimeManager[F[_]] {
-  def captureResults(
-      startHash: StateHash,
-      deploy: Signed[DeployData]
-  ): F[Seq[Par]]
   def replayComputeState(startHash: StateHash)(
+      rand: Blake2b512Random,
```

Comment:
Same here and in all other methods. Rholang terms should be first because they are main parameters which are executed and random generator immediately after.

I'm OK if you want to put random generator first but be consistent on all methods.